src/Cli/dotnet/Commands/Pack/PackCommand.cs

```diff
@@ -40,16 +46,78 @@ public static PackCommand FromParseResult(ParseResult parseResult, string? msbui
             CommonOptions.PropertiesOption,
             CommonOptions.RestorePropertiesOption,
             PackCommandParser.TargetOption);
+
         return new PackCommand(
             parsedMSBuildArgs.CloneWithAdditionalProperties(projectLocator.GetCustomDefaultConfigurationValueIfSpecified()),
             noRestore,
             msbuildPath);
     }
+    private static LogLevel MappingVerbosityToNugetLogLevel(VerbosityOptions? verbosity)
+    {
+        return verbosity switch
+        {
+            VerbosityOptions.diagnostic or VerbosityOptions.diag => LogLevel.Debug,
+            VerbosityOptions.minimal or VerbosityOptions.m => LogLevel.Minimal,
+            VerbosityOptions.normal or VerbosityOptions.n => LogLevel.Information,
+            VerbosityOptions.detailed or VerbosityOptions.d => LogLevel.Verbose,
+            _ => LogLevel.Minimal
+        };
+    }
+    public static int RunPackCommand(ParseResult parseResult)
```

Comment:
I'm wondering if the method that's here should actually be somewhere on the NuGet.Client side to allow us more easily iterate in the future?